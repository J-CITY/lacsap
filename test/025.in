a := 2 + i * i / i;
