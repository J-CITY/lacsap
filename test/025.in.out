a   0   0      identifier    24
:=   0   2       operation    22
2   0   5      literalInt    26
+   0   7       operation    20
i   0   9      identifier    24
*   0  11       operation     1
i   0  13      identifier    24
/   0  15       operation     2
i   0  17      identifier    24
;   0  18       separator     5
